src/components/Button/index.examples.js

```diff
@@ -3,9 +3,78 @@ import {storiesOf} from '@kadira/storybook'
 import {stringFixture, clickHandlerFixture} from '../../utils/Fixtures'
 import Button from '.'
 
+const standardButtonExamples = [
+  {type: 'default'},
+  {type: 'primary'},
+  {type: 'success'},
+  {type: 'warning'},
+  {type: 'danger'}
+]
+
+const dispStyles = 'flex flex-row content-center justify-center flex-wrap items-center vh-100 bg-navy'
+
+// Standard Large Button (Default)
 storiesOf('Button')
-  .addWithInfo('API', () => (
-    <Button onClick={clickHandlerFixture}> 
-      {stringFixture}
-    </Button>
+  .addWithInfo('Large (Default)', `The Large Default Button is available in several different flavors.`, () => (
+    <div className={dispStyles}>
+      {standardButtonExamples.map((btn, i) => {
+        return (
+          <div className='mb2 mh2' key={i}>
+            <Button type={btn.type} onClick={clickHandlerFixture}>
+              {btn.type}
+            </Button>
+          </div>
+        )
+      })}
+    </div>
   ))
+
+// Standard (Small) Button
+storiesOf('Button')
+  .addWithInfo('Small', `This Small Button is available in several different flavors.`, () => (
+    <div className={dispStyles}>
+      {standardButtonExamples.map((btn, i) => {
+        return (
+          <div className='mb2 mh2' key={i}>
```

Comment:
I would actually vote for not including extra presentational classes in the storybook examples; that way what you see is what you get. It wont look as pretty (components will be flush to the edges of the storybook pane) but that is how they are actually rendered when imported so it is more honest and free from side effects if composed together into layout component examples. See how the other `index.examples.js` were done and copy that.

Also, fyi, if you did want presentational stuff, the "blessed" way to do it is with the storybook decorators as documented here https://getstorybook.io/docs/react-storybook/basics/writing-stories